RELEASE.txt

```diff
@@ -177,6 +177,10 @@ access), specifically to:
    python setup.py sdist
    twine upload dist/scikit-image-<major>.<minor>.0.tar.gz
 
+- Publish the release on github:
+
+  Go to https://github.com/scikit-image/scikit-image/releases
```

Comment:
```suggestion
  Go to https://github.com/scikit-image/scikit-image/releases and
  click on "Draft a new release".
  Choose the tag corresponding to this release, copy the release notes,
  and upload the wheels in the appropriate slot. Finally, click on
  "Publish release".
```

Reply:
@sciunto what do you think of the above? It was not immediately obvious to me when I went to that URL. The button is not super prominent. =)

Reply:
My eyes caught that button at the first glance, but yes, nice addition! Explicit is better :)